setup.py

```diff
@@ -138,7 +138,8 @@ def run_tests(self):
       ext_modules=extensions,
       include_dirs=[np.get_include()],
       install_requires=install_requires,
-      extras_require={'hdf5': ["h5py >= 2.2.0"]
+      extras_require={'hdf5': ["h5py >= 2.2.0"],
```

Comment:
ah, this is vestigial... h5py is required. But, that does not need to be addressed in this PR>